plugins/cisco_umbrella_reporting/help.md

```diff
@@ -9,8 +9,11 @@ This plugin utilizes Cisco Umbrella to get the most complete view of the relatio
 
 # Requirements
 
-* Requires an API Key and Secret from Cisco Umbrella (Legacy API Key) - In the links section it's described how to create/refresh/delete it)
-* Requires Cisco Umbrella organization ID
+* Requires an API Key and Secret from Cisco Umbrella (API Key) - In the links section it's described how to create/refresh/delete it)
```

Comment:
Suggested reword- 'Refer to the links section for information on how to create/refresh/delete the API key.